packages/core/src/tools/heavyCustomerDataWarning.ts

```diff
@@ -0,0 +1,24 @@
+import { display } from './display'
+import { ONE_KIBI_BYTE } from './utils'
+
+// RUM batch bytes limit is 16KB
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:**
```suggestion
// RUM and logs batch bytes limit is 16KB
```